Forward the no_createrepo argument in add_rpm and delete_rpm to be consistent

lathama opened the pull request:
I am working on a repo server that tracks the number of times an rpm has been downloaded to better understand the usefulness of a version on a private repo. Decided to share some thoughts. Feel free to ignore.

lathama committed:
Update repo_manager.spec

lathama committed:
Duplicate the better delete / add functionality

lathama committed:
Address duplication from a change

Appears to have been a cut and double paste.

lathama commented:
Just noticed these comments. Addressed the duplication. The changelog reads strangely, why not bump the release if you want to keep the comments separate.

pypingou commented:
The changelog reads strangely because the change hasn't been released yet, when there will be the next release the date will be added above and it will read fine.

pypingou commented:
I still like this change (minus the change to the spec file). Would you like to update the PR for this?

lathama commented:
Sure sorry I missed this, few min

lathama committed:
Changelog revert

Changelog has unreleased changes above the first dated release to show what is not released.

lathama commented:
There fixed, will configure github to email me.

pypingou commented:
Thanks!

lathama commented:
email alerts work now! yeah
